### ISSUE TITLE
Enforce consistent `__slots__` syntax

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#379]
# Rule request

## Thesis

`__slots__` should always be `set`: this should be the only valid syntax:

```python
class Test(object):
     __slots__ = {'x', 'y'}
```

## Reasoning

This is done for consistency reasons.
`set` is selected because it can not contain duplicates and explicitly is not ordered.

**Comment by sobolevn** [comment]
Yes, this is a valid point. In this case we will need to check that slots are unique:

```python
>>> class Test:
...     __slots__ = ('a', 'a')
...
>>> Test()
<__main__.Test object at 0x10bbbce48>
```
